Baidak_Max/HW7/Task_3.py

```diff
@@ -0,0 +1,21 @@
+# Task_3
+# Write a function that calculates the number of characters included in given string
+
+
```

Comment:
good job
look at this implementation

def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result